PokeAlarm/Filters/BaseFilter.py

```diff
@@ -74,6 +75,18 @@ def evaluate_attribute(self, limit, eval_func, event_attribute):
         self._check_list.append(check)
         return limit
 
+    def evaluate_time(self, min_time, max_time):
```

Comment:
For unit-testing, you can optionally add a 3rd arg here `override_time` that defaults to `Null` then pass this argument as a 3rd arg to `CheckTime()` where you'd do the evaluation with the value of `override_time` being the time used instead of `datetime.now()`

Reply:
Tweaked something to override `current_time` for unit tests. Tests passed:
```
Ran 31 tests in 0.260s
OK
```